pkg/agent/route/route_windows.go

```diff
@@ -260,8 +260,8 @@ func (c *Client) addVirtualServiceIPRoute(isIPv6 bool) error {
 
 // TODO: Follow the code style in Linux that maintains one Service CIDR.
 func (c *Client) addServiceRoute(svcIP net.IP) error {
-	obj, found := c.hostRoutes.Load(svcIP.String())
 	svcIPNet := util.NewIPNet(svcIP)
+	obj, found := c.hostRoutes.Load(svcIPNet.String())
```

Comment:
L309 also needs update, can we add unit tests to cover them?

Reply:
> L309 also needs update
Done

> can we add unit tests to cover them?

Add some test code to verify the cached route in `c.hostRoutes`